main/src/main/assets/full_licenses.html

```diff
@@ -279,7 +279,7 @@ <h3> Original SSLeay License</h3>
 <br> * OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF
 <br> * SUCH DAMAGE.
 <br> * 
-<br> * The licence and distribution terms for any publically available version or
+<br> * The licence and distribution terms for any publicly available version or
```

Comment:
These probably get dropped in the long run, in addition to being removed from this PR.